### ISSUE TITLE
display labels on templates

### ISSUE THREAD

**Issue by ebb-tide** [influxdata/influxdb#12907]
Currently, labels on templates are not displayed. They should be displayed.

**Comment by russorat** [comment]
this should not be blocked on #12962 in my opinion. We need the user to be able to add labels to variables just like any other resource. We can add system labels later.

**Comment by OfTheDelmer** [comment]
So digging into this the `DocumentCreate` accepts labels but the `DocumentUpdate` only accepts changes to the `meta` and `content` properties. It looks like we'll need a separate issue for adding/removing labels via a `/documents/templates/{id}/labels`

**Comment by ebb-tide** [comment]
yes ! https://github.com/influxdata/influxdb/issues/13064

**Comment by OfTheDelmer** [comment]
@ebb-tide
